### ISSUE TITLE
Report

### ISSUE THREAD

**Issue by psaegert** [MrWhatZitToYaa/IGNNS-final-project#37]
https://www.overleaf.com/1665333581xdnzdrkbsrbh#f98d6e

- [x] Abstract
- [x] Introduction @CKleiber 
- [x] Project Planning @MrWhatZitToYaa 
- [x] Background
  - [x] NF @CKleiber (possibly)
  - [ ] ~~LSTM~~
  - [x] CNN @MrWhatZitToYaa 
- [x] Simulation & Data @CKleiber 
  - [x] Simulation @CKleiber 
    - [x] Physics @CKleiber 
    - [x] Cameras @CKleiber 
    - [x] Priors and Acceptance @CKleiber 
  - [x] Real Video Examples @CKleiber 
    - [x] Gathering Parameters @CKleiber 
    - [x] Video Processing @CKleiber 
- [x] Methods
  - [x] CNF Architecture @psaegert 
  - [x] Feature Network Architecture @MrWhatZitToYaa 
  - [x] Time Series Network @psaegert 
  - [x] Trainer and Wandb @MrWhatZitToYaa 
  - [x] Data movement and resource utilisation optimisation @MrWhatZitToYaa 
  - [x] Training @psaegert 
    - [x] INN NLL loss @psaegert 
    - [x] TrainDivergenceError @psaegert 
    - [x] ReduceLROnPlateau Scheduler @psaegert 
    - [x] Gradient clipping @psaegert 
    - [x] Convergence: @psaegert 
      - [x] max epochs @psaegert 
      - [x] timeout @psaegert 
      - [x] val loss plateau @psaegert 
  - [x] Hyperparameter Optimization @psaegert 
  - [x] Evaluation
    - [x] NLL Loss @psaegert 
    - [x] Posterior Distribution (qualitative) @psaegert 
    - [x] Calibration (qualitative) @psaegert 
    - [x] Resimulation (qualitative & quantitative) @psaegert 
      - [x] Plots @psaegert 
      - [x] Median Resimulation Error over time @psaegert 
- [x] Experiments
  - [x] Pilot Tests @psaegert 
  - [x] Hyperparameter Optimization @psaegert 
  - [x] Trajectory: FCNN, LSTM, Transformer @psaegert  
  - [x] Try to imporve training speed with Frankensteins monster @MrWhatZitToYaa 
  - [x] Double the CNN so there are is one for each camera @MrWhatZitToYaa 
- [x] Results
  - [x] Hyperparameter Optimization @psaegert 
  - [x] Trajectory: FCNN, LSTM, Transformer  @psaegert 
    - [x] Losses @psaegert 
    - [x] Calibration @psaegert 
    - [x] Resimulation @psaegert 
    - [x] Point of Impact Metrics @psaegert 
    - [x] Resimulation Error @psaegert 
  - [x] Video
  - [x] Video IRL
- [x] Conclusion
  - [x] Improving the performance of the CNN @MrWhatZitToYaa 
  - [x] Benefits of working as a team @MrWhatZitToYaa 
- [x] References
- [x] Appendix
  - [x] Priors
  - [x] Tools
  - [x] Cost @psaegert 
  - [x] Hyperparameter Results @psaegert 
  - [x] Trajectory Calibration @psaegert 
  - [x] Trajectory Impact @psaegert 
  - [x] Trajectory Resimulation @psaegert 
  - [x] Trajectory Resimulation Error @psaegert
